src/teaching-element/index.vue

```diff
@@ -77,17 +77,16 @@ export default {
   text-align: left;
 }
 
- div[class*="col-xs-"].te-container {
-   padding: 0;
- }
+div[class*="col-xs-"].te-container {
+  padding: 0;
+}
 
- .noselect {
+.noselect {
   -webkit-touch-callout: none; /* iOS Safari */
-    -webkit-user-select: none; /* Safari */
-     -khtml-user-select: none; /* Konqueror HTML */
-       -moz-user-select: none; /* Firefox */
-        -ms-user-select: none; /* Internet Explorer/Edge */
-            user-select: none; /* Non-prefixed version, currently
-                                  supported by Chrome and Opera */
+  -webkit-user-select: none; /* Safari */
+  -khtml-user-select: none; /* Konqueror HTML */
+  -moz-user-select: none; /* Firefox */
+  -ms-user-select: none; /* Internet Explorer/Edge */
+  user-select: none; /* Non-prefixed version, currently supported by Chrome and Opera */
```

Comment:
This is exactly why we have autoprefixer. 🙃

Reply:
This was linted, boi. It was already there. 💋